cli/src/main/kotlin/commands/RequirementsCommand.kt

```diff
@@ -100,12 +100,21 @@ object RequirementsCommand : CommandWithHelp() {
         allTools.forEach { category, tools ->
             println("${category}s:")
             tools.forEach { tool ->
-                if (tool.getVersionRequirement().toString() == anyVersion) {
-                    println("\t${tool.javaClass.simpleName} requires '${tool.command()}' in no specific version.")
+                val message = if (tool.getVersionRequirement().toString() == anyVersion) {
+                    "${tool.javaClass.simpleName} requires '${tool.command()}' in no specific version."
                 } else {
-                    println("\t${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
-                            "'${tool.command()}'.")
+                    "${tool.javaClass.simpleName} has ${tool.getVersionRequirement()} on the version of " +
+                            "'${tool.command()}'."
                 }
+
+                // TODO: State which version was found, but this requires refactoring of CommandLineTool.
```

Comment:
We should also state if a tool can be bootstrapped, if not found.

Reply:
Done.